lstchain/tests/test_lstchain.py

```diff
@@ -75,58 +86,51 @@ def test_import_lstio():
 @pytest.mark.run(order=1)
 def test_dl0_to_dl1():
     from lstchain.reco.dl0_to_dl1 import r0_to_dl1
-    infile = get_dataset_path('gamma_test_large.simtel.gz')
-    r0_to_dl1(infile, custom_config=custom_config)
+    infile = get_dataset_path(testfile)
+    r0_to_dl1(infile, custom_config=custom_config, output_filename=dl1_file)
 
 @pytest.mark.run(order=2)
 def test_build_models():
     from lstchain.reco.dl1_to_dl2 import build_models
-    infile = 'dl1_gamma_test_large.h5'
+    infile = dl1_file
 
     reg_energy, reg_disp, cls_gh = build_models(infile, infile, custom_config=custom_config, save_models=True)
 
     from sklearn.externals import joblib
-    joblib.dump(reg_energy, 'rf_energy.pkl')
-    joblib.dump(reg_disp, 'rf_disp.pkl')
-    joblib.dump(cls_gh, 'rf_cls_gh.pkl')
+    joblib.dump(reg_energy, os.path.join(test_dir, 'rf_energy.pkl'))
```

Comment:
Models' names could be defined also as variables.

Reply:
Indeed, thanks for the suggestion.